extensions-core/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureAccountConfig.java

```diff
@@ -94,9 +108,13 @@ public String getSharedAccessStorageToken()
     return sharedAccessStorageToken;
   }
 
-  @SuppressWarnings("unused") // Used by Jackson deserialization?
-  public void setSharedAccessStorageToken(String sharedAccessStorageToken)
+  public String getEndpointSuffix()
   {
-    this.sharedAccessStorageToken = sharedAccessStorageToken;
+    return endpointSuffix;
+  }
+
+  public String getBlobStorageEndpointSuffix()
```

Comment:
isn't it the whole endpoint itself? why are we calling this method `getBlobStorageEndpointSuffix` instead of `getBlobStorageEndpoint`

Reply:
Renamed